### PR TITLE
Add transcript id to the fasta headers to ensure uniqueness

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

**Comment by ebelter** [comment]
+1

File: lib/perl/Genome/VariantReporting/Suite/Vep/EpitopeVariantSequenceInterpreter.pm [hunk 1]
```diff
@@ -40,10 +40,17 @@ sub _interpret_entry {
...
+    my %transcript_count;
     ALLELE: for my $variant_allele (@$passed_alt_alleles) {
         my @transcripts = $vep_parser->transcripts($entry, $variant_allele);
 
         TRANSCRIPT: for my $transcript (@transcripts) {
+            if (defined($transcript_count{$transcript->{feature}})) {
+                $transcript_count{$transcript->{feature}} += 1;
+            }
+            else {
+                $transcript_count{$transcript->{feature}} = 1;
```
Comment (tmooney) [inline lib/perl/Genome/VariantReporting/Suite/Vep/EpitopeVariantSequenceInterpreter.pm]: `$transcript_count{$transcript->{feature}}++` covers both of these cases.

Comment (susannasiebert) [inline lib/perl/Genome/VariantReporting/Suite/Vep/EpitopeVariantSequenceInterpreter.pm]: Fixed. Thanks for the tip.

**Comment by tmooney** [comment]
+1 :bowling:

**Pull request merged**